components/Header/Header.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
rename this folder to lowercase `header` instead of `Header` for consistency with other components

Reply:
Question for our 1 on 1 - on local, showing up as lowercase header dir. Also, in welcomePage.js (main page) -> importing Header as lowercase e.g. ../components/header/Header. Why's it still in lowercase on this pr?

Reply:
Resolved, thanks to your solution: git config core.ignorecase false.